### DISCUSSION TITLE
Rancher Desktop 1.4.1

### DISCUSSION THREAD

**Discussion by jandubois** [rancher-sandbox/rancher-desktop#2351] [General]
This is the 1.4.1 release of Rancher Desktop, an open source desktop application to bring Kubernetes and container management to Mac, Windows, and Linux. The release contains some new features along with various bug fixes.

## Important Updates

* Rancher Desktop now ships with registry credential helpers and configures them automatically: It uses `pass` on Linux, `osxkeychain` on macOS, and `wincred` on Windows. The `ecr-login` helper is also included but not automatically configured.

  For `nerdctl` (which has to run inside the VM for technical reasons) a credential forwarder will redirect all credential requests to the helper on the host, so no credentials are installed inside the VM.

* The Images list now supports bulk deletion of images.

* Various bug fixes to the still experimental host resolver (DNS). We expect to be able to make it the default resolver in the 1.5.0 release.

## Updated utilities

- `docker` has been upgraded to 20.10.16.
- `docker-buildx` has been upgraded to 0.8.2.
- `docker-compose` has been upgraded to 2.5.1.
- `nerdctl` has been upgraded to 0.20.0.
- `helm` has been upgraded to 3.9.0.
- `trivy` has been upgraded to 0.28.0.

You can connect with the developers via:

- The [issue queue](https://github.com/rancher-sandbox/rancher-desktop/issues)
- [Rancher Users Slack](https://slack.rancher.io/) in the #rancher-desktop channel

## Changelog

The [full version changelog, from v1.3.0, can be found using GitHub compare](https://github.com/rancher-sandbox/rancher-desktop/compare/v1.3.0...v1.4.1) and the details of the release can be found in the [v1.4.1 milestone](https://github.com/rancher-sandbox/rancher-desktop/milestone/18?closed=1) (1.4.1 is the first public release from the `release-1.4` branch).

<hr /><em>This discussion was created from the release <a href='https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.4.1'>Rancher Desktop 1.4.1</a>.</em>

**Comment by kipper3d** [comment]
How about some documentation on this registry credential helpers? I use .docker/config.json on my osx and has worked until 1.4.1 and now it doesnt. I can't find any documentation on how to utilize this 'registry credential helpers'. I can no longer connect automatically instead I have to use docker login <registry> repeatedly. Thanks.

**Reply by jandubois** [reply]
@kipper3d It still uses the same mechanism. E.g.

```json
{
  "auths": {
    "registry.example.com": {}
  },
  "credsStore": "osxkeychain"
}
```

The credentials are stored in the local keychain instead of directly inside `config.json`. You can find more details at https://github.com/docker/docker-credential-helpers.

Please file a Github issue with the details of your problem, and including the content of your `~/.docker/config.json` (but redact any confidential info first). You should only need to `docker login` once per registry.
